src/api/eth_subscribe.rs

```diff
@@ -42,7 +42,7 @@ impl From<String> for SubscriptionId {
 /// Stream of notifications from a subscription
 /// Given a type deserializable from rpc::Value and a subscription id, yields items of that type as
 /// notifications are delivered.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
I removed this clone, because DuplexTransport::NotificationStream isn't clonable (not sure how this worked with the derivation, but it's possible that I'm missing something)

Reply:
I suppose `derive` was adding `T::NotificationStream: Clone` bound as well, hence it worked.